test/src/edit/DistortableCollectionEditSpec.js

```diff
@@ -123,8 +123,9 @@ describe('L.DistortableCollection.Edit', function () {
             expect(map._toolbars).to.be.empty;
 
             // need both bc simulated `mousedown`s don't fire `click` events afterwards like regular user generated `mousedown`s.
-            chai.simulateShiftMousedown(overlay.getElement());
-            chai.simulateClick(overlay.getElement());
+            chai.simulateEvent(overlay.getElement(), chai.mouseEvents.ShiftDown);
+            chai.simulateEvent(overlay.getElement(), chai.mouseEvents.ShiftDown);
```

Comment:
```suggestion

```
I think you accidentally added this line twice!! If so, lets delete the extra one